openquake/server/views.py

```diff
@@ -276,6 +277,33 @@ def validate_nrml(request):
         return _make_response(error_msg=None, error_line=None, valid=True)
 
 
+@csrf_exempt
+@cross_domain_ajax
+@require_http_methods(['POST'])
+def validate_zip(request):
+    """
+    Leverage the engine libraries to check if a given job.zip file is valid
+
+    :param request:
+        a `django.http.HttpRequest` object containing a zip archive
+
+    :returns: a JSON object, containing:
+        * 'valid': a boolean indicating if the provided job.zip is valid
+        * 'error_msg': the error message, if any error was found
+                       (None otherwise)
+    """
+    archive = request.FILES.get('archive')
+    if not archive:
+        return HttpResponseBadRequest('Missing job.zip file')
```

Comment:
``job.zip`` is too specific without a real reason.

Reply:
Changed to archive